src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Helpers.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Text.Json.Serialization;
+using System.Text.Json.Serialization.Metadata;
+
+namespace System.Text.Json
+{
+    public static partial class JsonSerializer
+    {
+        private static JsonTypeInfo GetTypeInfo(Type runtimeType, JsonSerializerOptions? options)
```

Comment:
Curious, what is the motivation for moving those helpers methods to the main public class?

Reply:
We have existing read/write-specific non-public `JsonSerializer` helpers on this public class, but none that apply to both reading and writing (shared). We've been writing shared helpers on `JsonHelpers`, but that seems more appropriate for helpers shared across all the types (reader, writer, serializer, documents).